### PR TITLE
Go generate

### PR DESCRIPTION
**Pull request by clrprod**
Convert more files over to go generate.

### CODE REVIEW

File: protoc/generate.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//go:generate rm -rf include
```
Comment (aysylu) [inline protoc/generate.go]: What's the purpose of these commented out lines?

Comment (clrprod) [reply protoc/generate.go]: That's how you configure go generate. It runs each line in order from within the folder.

Comment (noahklein) [reply protoc/generate.go]: I think a comment at the top explaining what these are generating, would be helpful. e.g. "Generates the protos and proto compilation tools."

Comment (aysylu) [reply protoc/generate.go]: Anything that explains that these commented-out lines actually have an effect (and optionally, adding what effect, at high-level), across all generate.go files would be helpful. Copy/paste is fine.

Comment (clrprod) [reply protoc/generate.go]: I added a comment to each of the generate.go files, let me know if you want more explanation.

**Pull request merged**